prprocessor/__main__.py

```diff
@@ -373,18 +367,42 @@ async def on_pr_merge(*, pull_request: Mapping, **_kw) -> None:
     if issue_ids:
         redmine = get_redmine()
         project = redmine.project.get(config.project)
-        fixed_in_version = get_latest_open_version(project, version_prefix)
-
-        if not fixed_in_version:
-            logger.info('Unable to determine latest version for %s; prefix=%s', project.name,
-                        version_prefix)
-            return
-
-        for issue in get_issues(redmine, issue_ids):
-            if issue.project.id == project.id:
-                logger.info('Setting fixed in version for issue %s to %s', issue.id,
-                            fixed_in_version.name)
-                set_fixed_in_version(issue, fixed_in_version)
+
+        if pull_request['merged']:
+            target_branch = pull_request['base']['ref']
+            version_prefix = get_version_prefix_from_branch(target_branch)
+            if not version_prefix:
```

Comment:
`version_prefix` can be `''` which is `false`-y, but still a valid prefix, I guess you wanted `is not None`?